src/components/CustomDisciplineMenuItem.tsx

```diff
@@ -130,45 +78,21 @@ const CustomDisciplineMenuItem = ({
     )
   } else {
     return (
-      <View testID={'Swipeable'}>
-        <Swipeable
-          ref={updateRef}
-          friction={2}
-          enableTrackpadTwoFingerGesture
-          rightThreshold={40}
-          renderRightActions={renderRightAction}>
-          <ConfirmationModal
-            visible={isModalVisible}
-            setVisible={setIsModalVisible}
-            text={labels.home.deleteModal.confirmationQuestion}
-            confirmationButtonText={labels.home.deleteModal.confirm}
-            cancelButtonText={labels.home.deleteModal.cancel}
-            confirmationAction={deleteModule}
-          />
-          {data ? (
-            <MenuItem
-              selected={idToSelectedIdString(data.id) === selectedId}
-              onPress={() => {
-                setSelectedId(idToSelectedIdString(data.id))
-                navigation.navigate('DisciplineSelection', {
-                  extraParams: { discipline: data }
-                })
-              }}
-              icon={data.icon}
-              title={data.title}>
-              <Description selected={idToSelectedIdString(data.id) === selectedId}>
-                {data.numberOfChildren} {data.numberOfChildren === 1 ? labels.home.unit : labels.home.units}
-              </Description>
-            </MenuItem>
-          ) : (
-            <Placeholder>
-              <ErrorText>
-                {labels.home.errorLoadCustomDiscipline} {apiKey}
-              </ErrorText>
-            </Placeholder>
-          )}
-        </Swipeable>
-      </View>
+      <DeletionSwipable apiKey={apiKey} refresh={refresh}>
```

Comment:
Typo, please check the naming everywhere
```suggestion
      <DeletionSwipeable apiKey={apiKey} refresh={refresh}>
```